.github/scripts/create-release.sh

```diff
@@ -0,0 +1,194 @@
+#! /usr/bin/env bash
+
+## Description:
+## 
+## This script automates the process involved in creating a new release for the opendatahub-io/kubeflow repository as
+## documented in https://issues.redhat.com/browse/RHOAIENG-15391.
```

Comment:
we can mention that for local execution the `gh` cli tool and GH token need to be configured on the machine

Reply:
will add this in a subsequent PR - good call to include

Reply:
in general.. thinking of this as a "template" - a "requirements" section would be in general a good thing to always include in this top level comment 💯

Reply:
> mention that for local execution the gh cli tool and GH token need to be configured on the machine

Also check for both at the very beginning and fail if that's not present? Nobody ever reads the comments until they get such a strong indication they actually are gonna have to. It's always helpful if a script checks all prereqs first thing it does.